README.md

```diff
@@ -3,6 +3,14 @@
 Quantum software benchmarking
 ___
 
+## What is Benchpress?
+
+Benchpress is an open-source tool for benchmarking quantum software.
+
+The Benchpress open-source benchmarking suite comprises over 1,000 different tests. Although Benchpress is a package built by IBM, the tests themselves are standardized benchmarking tests designed by other members of the quantum community. For example, Benchpress compares SDKs’ abilities to generate [QASMBench circuits](https://github.com/pnnl/QASMBench), [Feynman circuits](https://github.com/meamy/feynman), and [Hamiltonian circuits](https://arxiv.org/pdf/2306.13126](https://portal.nersc.gov/cfs/m888/dcamps/hamlib/). It also includes tests designed to test a language's ability to transpiler circuits for specific hardware, including the heavy hex architecture of IBM quantum processors and other generic qubit layouts.
```

Comment:
I am not sure that I am a fan of the "built by IBM" piece.  While this is true now, it goes away with community involvement, and we would have to remember to remove it.  It also feels like we are trying to justify things that need no justification.

Reply:
@nonhermitian I'm happy to remove it. It was there simply to match over-cautious language from the blog but I suppose it's not nearly as important here.